src/main/java/com/raisetech/cafeinfo/CafeService.java

```diff
@@ -0,0 +1,29 @@
+package com.raisetech.cafeinfo;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CafeService {
+
+    public CafeMapper cafeMapper;
+
+    public CafeService(CafeMapper cafeMapper) {
+        this.cafeMapper = cafeMapper;
+    }
+
+    List<Cafe> findAll() {
+        return cafeMapper.getAll();
+    }
+
+    public List<Cafe> findByPlaces(String place) {
+        return cafeMapper.findByPlaceWith(place);
+    }
+
+    public Cafe findId(int id) {
```

Comment:
```suggestion
    public Cafe findById(int id) {
```
findIdだと「IDをみつける」という意味になりますね。

Reply:
かしこまりました。

適切な名称に変更いたしました。
ご確認のほどよろしくお願いいたします。
0c4481dfaa86697dae95ab694466e338dabce77a